README.md

```diff
@@ -65,11 +65,13 @@ module.exports = function fetch (callback) {
 ```
 
 ```js
-var proxyquire = require('proxyquire');
-var fetch = proxyquire('./get', {
+var assert = require('assert');
+var proxyquire = require('proxyquire').noCallThru();
```

Comment:
Undo this change and require proxyquire first

Reply:
I do think that adding `noCallThru` is really essential because I didn't have `simple-get` installed and it was failing and I had no clue why.